scripts/create_elmo_embeddings_from_vocab.py

```diff
@@ -0,0 +1,87 @@
+# pylint: disable=no-self-use
+
+import os
+import argparse
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir))))
+
+import h5py
+import numpy
+import torch
+from torch.autograd import Variable
+
+from allennlp.data.token_indexers import ELMoTokenCharactersIndexer
+from allennlp.modules.elmo import _ElmoCharacterEncoder
+from allennlp.data import Token, Vocabulary
+
+
+def main(vocab_path: str,
+         elmo_model_dir: str,
+         output_dir: str,
+         batch_size: int,
+         device: int):
+
+    options_file = os.path.join(elmo_model_dir, 'options.json')
+    weight_file = os.path.join(elmo_model_dir, 'lm_weights.hdf5')
+
+    # Load the vocabulary words and convert to char ids
+    with open(vocab_path, 'r') as vocab_file:
+        tokens = vocab_file.read().strip().split('\n')
```

Comment:
We need to add the special tokens `<S>` and `</S>` to this file if they don't exist.  Are there other special tokens necessary to make it usable with the allennlp Vocabulary?  Also, do we need to modify the file format so that it can be loaded with an allennlp Vocabulary?